tst/SdpApiTest.cpp

```diff
@@ -811,7 +848,9 @@ a=rtcp-mux
 a=rtcp-rsize
 a=rtpmap:96 VP8/90000
 a=rtpmap:102 H264/90000
-a=fmtp:102 strange
```

Comment:
Change: the logic no longer accepts an H264 fmtp value without exact required characteristics.